app/cafe.py

```diff
@@ -0,0 +1,18 @@
+from datetime import date
+from app.errors import NotWearingMaskError, \
+    OutdatedVaccineError, \
+    NotVaccinatedError
```

Comment:
Bad import code style, better use:

```suggestion
from app.errors import (
    NotWearingMaskError, 
    OutdatedVaccineError, 
    NotVaccinatedError,
)
```